core/services/gas/optimism_estimator.go

```diff
@@ -296,7 +296,7 @@ func (o *optimism2Estimator) refreshPrice() (t *time.Timer) {
 	return
 }
 
-func (o *optimism2Estimator) OnNewLongestChain(_ context.Context, _ eth.Head) {}
+func (o *optimism2Estimator) OnNewLongestChain(ctx context.Context, head *eth.Head) {}
```

Comment:
Does your editor automatically fill these for you?

Reply:
Yeah, I did a GoLand signature change - will clean up.